app/assets/stylesheets/articles.scss

```diff
@@ -592,166 +321,105 @@
-        padding-top: 20px;
-      }
       p {
         margin: 6px auto;
         line-height: 20px;
         font-size: 16px;
-        width: 94%;
         color: rgb(163, 188, 201);
-        padding-bottom: 19px;
+        padding-bottom: 16px;
       }
       &.big-article {
         padding: 0px;
-        margin: 10px auto 0px;
-        @media screen and (min-width: 650px) {
-          margin: 15px auto 0px;
-        }
-        @media screen and (min-width: 950px) {
-          margin: 10px auto 0px;
-        }
         .picture {
-          width: 100%;
-          margin: auto;
           padding-top: 42%;
-          position: relative;
-          z-index: 0;
-          border-top-left-radius: 3px;
-          border-top-right-radius: 3px;
-
+          border-radius: 3px 3px 0 0;
           background: no-repeat center center;
           background-size: cover;
         }
         .content-wrapper {
-          @include themeable(background, theme-container-background, #fff);
-          @include themeable(color, theme-container-color, $black);
-          position: relative;
-          z-index: 5;
-          border-top: 1px solid darken($lightest-gray, 4%);
-          @include themeable(
-            border-top,
-            theme-container-border,
-            1px solid darken($lightest-gray, 4%)
-          );
-          @include themeable(
-            border-bottom,
-            theme-container-border,
-            1px solid darken($lightest-gray, 4%)
-          );
-          padding-bottom: 105px;
-          border-radius: 3px;
+          padding-bottom: 115px;
           h3 {
-            font-size: calc(1.7vw + 22px);
-            font-weight: 500;
-            width: calc(100% - 20px);
-            margin-left: 12px;
-            margin-bottom: 8px;
-            margin-top: 8px;
-            padding-top: 0px;
+            font-size: 28px;
+            padding: 16px;
+            margin: 0;
             word-break: break-word;
-            @media screen and (min-width: 1600px) {
-              font-size: 45px;
+
+            @media screen and (min-width: 450px) {
+              font-size: 40px;
             }
           }
           .title-block {
             display: inline-block;
           }
-          @media screen and (min-width: 430px) {
-            padding-bottom: 120px;
-          }
         }
       }
     }
     .featured-profile-button {
       float: left;
-      margin-top: -104px;
       z-index: 10;
-      left: 12px;
+      left: 16px;
       position: absolute;
-      @media screen and (min-width: 430px) {
-        margin-top: -124px;
-      }
+      margin-top: -110px;
     }
     img.featured-profile-pic {
       height: 40px;
       width: 40px;
       border-radius: 50px;
-      @media screen and (min-width: 430px) {
-        height: 50px;
-        width: 50px;
-      }
       &:hover {
         opacity: 0.9;
       }
     }
     .featured-user-name {
       font-weight: bold;
-      margin-top: -101px;
```

Comment:
Any idea why this was `-101px`?

Reply:
This is what you end up with when you want to position everything absolutely :D  I'm pretty sure this is the reason for that. We have/had more margins like that in feed card. 

Hopefully, we can rewrite the frontend code when working on new feed cards.